sdcm/nemesis.py

```diff
@@ -416,7 +416,7 @@ def disrupt_nodetool_drain(self):
             self.target_node.start_scylla_server(verify_up=True, verify_down=False)
 
     @retrying(n=3, sleep_time=60, allowed_exceptions=(NodeSetupFailed, NodeSetupTimeout))
-    def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=10800):
+    def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=18000):
```

Comment:
we said that could hide real issues, when running on smaller loads... why not to increase the "base" timeout if the data set will be bigger than 200GB?